scss/atoms/typography/_typography.scss

```diff
@@ -54,7 +54,7 @@ h6 {
 	font-size: em( 16px );
 	font-weight: 600;
 	line-height: 1;
-	text-transform: uppercase;
+	text-transform: uppercase; // FIXME Is this opinionated?
 }
```

Comment:
okay, then ill have that variable only for h6 and h5, we dont want the entire stylesheet in the variables